app/views/shared/_reservations_table.html.haml

```diff
@@ -24,6 +24,10 @@
             = link_to "#{od.product.facility.abbreviation} / #{product_name}".html_safe, facility_instrument_path(od.product.facility, od.product)
             - if od.note.present?
               .order-detail-extra.order-detail-note= "Note: #{od.note}"
+            - if od.product.respond_to?(:online?)
```

Comment:
What would you think of an `online?` method on `Product` that always responds to `true`?